example/project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject lein-doo-example "0.1.10-SNAPSHOT"
+(defproject lein-doo-example "0.2.1-SNAPSHOT"
```

Comment:
I'd prefer it if feature pull requests didn't change the version numbers – I'd like to do it separately when I do a new release and the next version is not going to be `0.2.1` anyway (it's either `0.1.10` or `0.2.0`). Could you remove these changes from your commit?